trunner/runners/Stm32l4Runner.py

```diff
@@ -0,0 +1,77 @@
+#
+# Phoenix-RTOS test runner
+#
+# Stm32l4 runner
+#
+# Copyright 2021 Phoenix SYstems
+# Authors: Mateusz Niewiadomski
+#
+import subprocess
+import serial
+import time
+import pexpect
+import pexpect.fdpexpect
+
+from .common import DeviceRunner
+from .common import _BOOT_DIR
+
+
+class Stm32l4Runner(DeviceRunner):
+
+    class oneByOne_fdspawn(pexpect.fdpexpect.fdspawn):
+
+        def send(self, s):
+            for c in s:
+                super().send(c)
+                time.sleep(0.03)
+
+        def sendline(self, s):
```

Comment:
should not be needed, the original implementation uses send() internally also

Reply:
Yes this is redundant redeclaration. Removed `sendline()` from `oneByOne_fdspawn` and it works well